tutorials/gitlab-instant-app/index.mdx

```diff
@@ -36,9 +36,9 @@ GitLab is a web-based Git repository manager with wiki and issue tracking featur
 
 ## Opening the mail ports to send e-mails
 
-To avoid spam, remote mail ports cannot be reached from our infrastructure by default. As GitLab requires to send mail, we need to open them. Your account needs to be validated to perform this operation. [Learn more about identiy verification](https://www.scaleway.com/en/docs/console/my-account/how-to/verify-identity).
+To avoid spam, remote mail ports cannot be reached from our infrastructure by default. As GitLab requires to send mail, we need to open them. Your account needs to be validated to perform this operation. [Learn more about identiy verification](https://www.scaleway.com/docs/console/my-account/how-to/verify-identity).
```

Comment:
```suggestion
To avoid spam, remote mail ports cannot be reached from our infrastructure by default. As GitLab requires to send mail, we need to open them. Your account needs to be validated to perform this operation. [Learn more about identiy verification](/console/my-account/how-to/verify-identity).
```